packages/hooks/__tests__/use-timeout.vitest.ts

```diff
@@ -20,8 +20,9 @@ describe('use-timeout', () => {
     wrapper = _mount(cb)
   })
   afterEach(() => {
-    vi.useRealTimers()
     vi.clearAllMocks()
+    vi.restoreAllMocks()
```

Comment:
I think `clearAllMocks` and `restoreAllMocks` are essentially functionally duplicated, can we only keep one of them?

Reply:
Fixed